core/src/main/java/org/elasticsearch/action/get/GetRequest.java

```diff
@@ -61,7 +61,6 @@
 
     private VersionType versionType = VersionType.INTERNAL;
     private long version = Versions.MATCH_ANY;
-    private boolean ignoreErrorsOnGeneratedFields;
```

Comment:
Note to any other reviewers: this is no longer supported. I haven't dug too deep but my guess is that it was removed when we removed GET from the translog.